submissions/sprint_b/app.py

```diff
@@ -0,0 +1,23 @@
+from sprint_b import app, jsonify
+from sprint_b.helpers import EpithetGenerator, Vocabulary
+
+
+@app.route('/')
+def generate_epithet():
+    epithet = EpithetGenerator.generate_epithet()
+    return jsonify({"epithet": epithet})
+
+
+@app.route('/epithets/<quantity>')
+def generate_epithets(quantity):
+    quantity = int(quantity)
```

Comment:
You can avoid typecasting yourself by letting Flask do it from the route declaration using '/epithets/<int:quantity>'